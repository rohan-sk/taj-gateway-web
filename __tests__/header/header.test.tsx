import React from "react";
import { render } from "@testing-library/react";
import Header from "../../components/header/header.component";

const data = [
  {
    logo: "image-eaf9ae941361d173fdbd613ba600197b69471d50-100x88-png",
    detailList: [
      { title: "DESTINATIONS" },
      { title: "HOTELS" },
      { title: "REWARDS" },
      { title: "DINING" },
      { title: "HOLIDAYS" },
      { title: "MORE" },
    ],
    loginList: [{ title: "LOGIN" }, { title: "EN" }],
    primaryAction: [{ title: "BOOK A STAY" }],
  },
];

it("shows header", () => {
  const { asFragment } = render(<Header {...data} />);
  expect(asFragment()).toMatchSnapshot();
});
