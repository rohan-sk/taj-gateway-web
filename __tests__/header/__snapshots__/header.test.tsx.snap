// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows header 1`] = `
<DocumentFragment>
  <nav
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-nwbw3v-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
    >
      <div
        class="MuiBox-root css-15zdsri"
      >
        <div
          class="MuiBox-root css-1w54twj"
        />
        <div
          class="MuiBox-root css-j4hctk"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-dqscbn-MuiSvgIcon-root"
            data-testid="MenuSharpIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
            />
          </svg>
        </div>
      </div>
    </div>
  </nav>
</DocumentFragment>
`;
